resources/lang/en/general.php

```diff
@@ -1,244 +1,254 @@
-    'license'				=> 'License',
+    'license'               => 'License',
     'license_report'        => 'License Report',
-    'licenses_available'	=> 'licenses available',
-    'licenses'				=> 'Licenses',
-    'list_all'				=> 'List All',
-    'loading'				=> 'Loading... please wait....',
+    'licenses_available'    => 'licenses available',
+    'licenses'              => 'Licenses',
+    'list_all'              => 'List All',
+    'loading'               => 'Loading... please wait....',
     'lock_passwords'        => 'This field value will not be saved in a demo installation.',
     'feature_disabled'      => 'This feature has been disabled for the demo installation.',
     'location'              => 'Location',
-    'locations'				=> 'Locations',
-    'logo_size'				=> 'Square logos look best with Logo + Text. Logo maximum display size is 50px high x 500px wide.  ',
-    'logout'				=> 'Logout',
-    'lookup_by_tag'     => 'Lookup by Asset Tag',
+    'locations'             => 'Locations',
+    'logo_size'             => 'Square logos look best with Logo + Text. Logo maximum display size is 50px high x 500px wide.  ',
+    'logout'                => 'Logout',
+    'lookup_by_tag'         => 'Lookup by Asset Tag',
     'maintenances'          => 'Maintenances',
-    'manufacturer'			=> 'Manufacturer',
-    'manufacturers'			=> 'Manufacturers',
-    'markdown'				=> 'This field allows <a href="https://help.github.com/articles/github-flavored-markdown/">Github flavored markdown</a>.',
-    'min_amt'				=> 'Min. QTY',
+    'manufacturer'          => 'Manufacturer',
+    'manufacturers'         => 'Manufacturers',
+    'markdown'              => 'This field allows <a href="https://help.github.com/articles/github-flavored-markdown/">Github flavored markdown</a>.',
+    'min_amt'               => 'Min. QTY',
     'min_amt_help'          => 'Minimum number of items that should be available before an alert gets triggered. Leave Min. QTY blank if you do not want to receive alerts for low inventory.',
-    'model_no'				=> 'Model No.',
-    'months'				=> 'months',
-    'moreinfo'				=> 'More Info',
-    'name'					=> 'Name',
-    'next'					=> 'Next',
-    'next_audit_date'		=> 'Next Audit Date',
-    'last_audit'		    => 'Last Audit',
-    'new'					=> 'new!',
-    'no_depreciation'		=> 'No Depreciation',
-    'no_results'			=> 'No Results.',
-    'no'  					=> 'No',
-    'notes'  				=> 'Notes',
+    'model_no'              => 'Model No.',
+    'months'                => 'months',
+    'moreinfo'              => 'More Info',
+    'name'                  => 'Name',
+    'next'                  => 'Next',
+    'next_audit_date'       => 'Next Audit Date',
+    'last_audit'            => 'Last Audit',
+    'new'                   => 'new!',
+    'new_asset'             => 'New Asset',
+    'new_accessory'         => 'New Accessory',
+    'new_consumable'        => 'New Consumable',
+    'new_license'           => 'New License',
```

Comment:
`new_asset`, `new_accessory`, `new_consumable`, and `new_license` are new phrases